git-repo-manager/src/service.rs

```diff
@@ -39,8 +47,16 @@ impl GitRepoManager for GitRepoManagerService {
             self.config.git_repository_base_path, request_data.repository_path
         );
 
+        // Log
+        info!("Creating new repository : {}", new_repo_full_path);
```

Comment:
Sometimes you put uppercase letters, sometimes you don't. If we had to chose, I think we shouldn't put any. What's your opinion on that ?

Reply:
I also think it's better to not capitalize on the start of the error message

Reply:
Both golang, pretty_env_logger and tracing use lowercase messages. I'd say it's more standard to use lowercase messages.

Reply:
Agree with you guys